packages/cli/generation/local-generation/local-workspace-runner/src/runLocalGenerationForWorkspace.ts

```diff
@@ -45,7 +45,8 @@ export async function runLocalGenerationForWorkspace({
                         irVersionOverride: generatorInvocation.irVersionOverride,
                         outputVersionOverride: undefined,
                         writeUnitTests: false,
-                        generateOauthClients: false
+                        generateOauthClients: false,
+                        generatePaginatedClients: false
```

Comment:
do you know if we made this `optional` or `required` so that the cli can run older versions of the docker that don't know about this feature flag

Reply:
in fiddle it's required, I can make it optional: https://github.com/fern-api/fiddle/pull/376/files

Reply:
here's that PR: https://github.com/fern-api/fiddle/pull/377

Reply:
pushed optional change in, resolving this thread!